pydm/data_plugins/__init__.py

```diff
@@ -15,6 +16,36 @@
 plugin_modules = {}
 
 
+DEFAULT_PROTOCOL = os.getenv("PYDM_DEFAULT_PROTOCOL")
+if DEFAULT_PROTOCOL is not None:
+    DEFAULT_PROTOCOL = DEFAULT_PROTOCOL.replace('://', '')
+    logger.info("Using default PyDM protocol %s",
+                DEFAULT_PROTOCOL)
+
+
+def plugin_for_address(address):
+    """
+    Find the correct PyDMPlugin for a channel
+    """
+    # Check for a configured protocol
+    match = re.match('.*://', address)
```

Comment:
Probably should be a non-greedy match here (or not a regex, perhaps):
```
In [1]: re.match('.*://', 'a://b://c').group(0)
Out[1]: 'a://b://'

In [2]: re.match('.*?://', 'a://b://c').group(0)
Out[2]: 'a://'
```
I don't think there's anything restricting the latter form in pydm addressing, but I could be wrong...

Reply:
Addressed on the last commit 👍